keras_cv/visualization/cam.py

```diff
@@ -0,0 +1,40 @@
+import tensorflow as tf
+
+
+@tf.function(experimental_relax_shapes=True, experimental_follow_type_hints=True)
+def gradcam(
```

Comment:
Is the file name `cam.py` or should be `grad_cam.py`?

Reply:
or, we're going to have also variants of cam, ie. `grad_cam_pluss_plus`, and `guided_gram` func in the single file? That's fine then.

Reply:
It could go whichever way, just let me know. I usually place them in the same file in my research projects, as these methods aren't usually very long.

Reply:
Correction: I place all cam-based methods inside `explaining/cams.py` and all saliency-based into `explaining/saliency.py`, with `__init__.py` importing and exposing all members.